packages/canary-integration/src/create-ipfs.ts

```diff
@@ -2,21 +2,33 @@ import * as dagJose from 'dag-jose'
 import { path } from 'go-ipfs'
 import * as Ctl from 'ipfsd-ctl'
 import * as ipfsHttp from 'ipfs-http-client'
-import type { IPFS, Options } from 'ipfs-core'
+import type { Options } from 'ipfs-core'
+import { create } from 'ipfs-core'
 import getPort from 'get-port'
 import mergeOpts from 'merge-options'
+import type { IpfsApi } from '@ceramicnetwork/common'
```

Comment:
Is this not possible to import directly from js-ipfs yet?

Reply:
It is possible, but for the sake of uniformity, I retained `IpfsApi`, until it is gone everywhere at once.

Reply:
Relevant: https://github.com/ceramicnetwork/js-ceramic/issues/1937